tests/unit/test_dagascii.py

```diff
@@ -0,0 +1,28 @@
+import mock
+import os
+
+from dvc import dagascii
+from dvc.env import DVC_PAGER
+
+
+def test_less_pager_returned_when_less_found():
+    with mock.patch.object(os, "system") as m:
```

Comment:
you can use `mocker` fixture instead of using `mock` directly.

Reply:
done, thanks!